src/installers/BepisLoaderInstaller.ts

```diff
@@ -9,9 +13,22 @@ export class BepisLoaderInstaller implements PackageInstaller {
 
     async uninstall(args: InstallArgs) {
         /*
-        Do nothing on uninstall as BepisLoader for Resonite currently has two mod loaders. Both are required and so
-        uninstalling one / installing one after another would leave us in a broken state.
+        Resonite deals with two mod loaders. As it's currently the only game using BepisLoader, for now we'll hard-code
+        mod loader packages and we'll remove the appropriate folders. We'll only remove the renderer for now.
          */
+        const { profile } = args;
+        const modList = await ProfileModList.getModList(profile);
+        if (modList instanceof R2Error) {
+            throw modList;
+        }
+
+        const hasRendererModLoaderInstalled = modList.findIndex(value => value.getName() === 'ResoniteModding-BepInExRenderer') >= 0;
+        if (!hasRendererModLoaderInstalled) {
+            const rendererPath = profile.joinToProfilePath('Renderer');
+            await FileUtils.ensureDirectory(rendererPath);
+            await FileUtils.emptyDirectory(rendererPath);
+            await FsProvider.instance.rmdir(rendererPath);
```

Comment:
Nitpick: I've at some point added `FileUtils.recursiveRemoveDirectoryIfExists` which basically does the same thing and may or may not be more readable depending on how one thinks about these things.